ethereum/src/consensus.rs

```diff
@@ -357,17 +357,36 @@ impl<S: ConsensusSpec, R: ConsensusRpc<S>> Inner<S, R> {
 
         self.bootstrap(checkpoint).await?;
 
-        let current_period = calc_sync_period::<S>(self.store.finalized_header.beacon().slot);
-        let updates = self
+        let expected_current_slot = self.expected_current_slot();
+        let expected_current_period = calc_sync_period::<S>(expected_current_slot);
+        let mut bootstrap_period = calc_sync_period::<S>(self.store.finalized_header.beacon().slot);
```

Comment:
Maybe we should rename this since it no longer stays as the bootstrap period the whole time? Maybe `next_updates_fetch_period` or something along those lines?